bin/upgrade-code

```diff
@@ -18,11 +18,30 @@ foreach ($paths as $path) {
 }
 
 use SilverStripe\Upgrader\Console as Upgrader;
+use SilverStripe\Upgrader\Util\UpdateChecker;
 use Symfony\Component\Console\Application;
+use M1\Env\Parser;
+
+// Get Upgrader version from environment file
+$version = '1.4.0';
```

Comment:
Hardcoded values like this should at least be `const`. Might be ok since this is a one-off and should never change.

Reply:
We'll need to change it manually each time we do a release. This is for people using `composer global require silverstripe/upgrader`. Basically, I want people who run the upgrader via `composer glboal` to still get the version and to still get a warning when they need to update.

Ideally we wouldn't put that there at all. I had a look at trying to read it from the `composer.lock` file, but it sounded like too much trouble for the benefit.

I didn't make it a const because it's not something that should be available throughout the upgrader codebase.